### ISSUE TITLE
Multiple accessibility regressions with tabs in the 1.0.1 beta

### ISSUE THREAD

**Issue by pitermach** [mlemgroup/mlem#440]
**Describe the bug**
With the new beta on testflight, VoiceOver no longer speaks the tabs on the bottom correctly. To reproduce, turn VoiceOver on, then touch one of the tabs at the bottom. Previously, VoiceOver would speak both the label of the tab, as well as whether it was selected - IE. "Selected, Feeds tab, 1 of 5." With the current testflight beta, VoiceOver now just says "tab X of Y", but no longer speaks either the label of the tab as well as whether it's selected or not. This is a very serious regression since a blind user of the app can no longer figure out which tab is which and this also makes it impossible to activate the tabs for someone using the VOice Control accessibility feature.

Additionally, the tabs are incorrectly placed as the first elements on the screen for accessibility. To see what I mean, with VoiceOver on, tap with 4-fingers near the top of the screen, which should move to the first (IE topmost) element on screen, which should be the section heading or a back button. Instead, the first tab gains focus. If you then swipe right past the tabs you'll reach the actual first item on-screen. Likewise, if you tap with 4 fingers near the bottom, VoiceOver should move the focus to the last element on the screen which should be the rightmost tab, but instead it takes you to the bottom of whatever section is being displayed.

**Mlem Information:**
1.0.1 (432)

**Additional Context**
I'm running the iOS 17 beta, though this works correctly on the app store version so it's probably not a factor here.

**Comment by JakeShirley** [comment]
I have a PR up for the first bit.  I'll have to consult others on how to fix the tab indexing for the second part.

**Comment by pitermach** [comment]
That's great, the first part is definitely the more important one so getting this in before the update drops will be good. ☺️ THanks for looking into this.
